src/ts/ContentMetadataBuilder.ts

```diff
@@ -1,47 +1,30 @@
+import * as Conviva from '@convivainc/conviva-js-coresdk';
+
 type NoStringIndex<T> = { [K in keyof T as string extends K ? never : K]: T[K] };
 
 type ReservedContentMetadata = NoStringIndex<Conviva.ConvivaMetadata>;
 
-export type CustomContentMetadata = { [key: string]: string };
-
-export interface Metadata {
-  // Can only be set once
-  assetName?: string;
-
-  // Can only be set before playback started
-  viewerId?: string;
-  streamType?: Conviva.valueof<Conviva.ConvivaConstants['StreamType']>;
-  applicationName?: string;
-  custom?: CustomContentMetadata;
-  duration?: number;
-
-  // Dynamic
-  encodedFrameRate?: number;
-  defaultResource?: string;
-  streamUrl?: string;
-  framework?: string;
-  frameworkVersion?: string;
-}
+export type Metadata = Conviva.ContentMetadata & {
+  // TODO extend it with additional standard tags
```

Comment:
It will be done in a follow-up PR.

Reply:
Done in https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/108.